.changeset/afraid-cats-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": patch
+---
+
+Allow any space tokens to be used as margin bottom on heading component
```

Comment:
I think the change would be not to allow any margin to this component. So, by saying that and to avoid any issue with the code that already exists, we can set the marginBottom as space0 by default. I'm saying that because we are always setting marginBottom as "space0" because the component is setting margin when it shouldn't.